drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/renderers/BaseExpressionGridTheme.java

```diff
@@ -0,0 +1,186 @@
+
+    public static final String GRID_STROKE_COLOUR = "#dbdbdb";
+
+    public static final String ROW_NUMBER_BACKGROUND_FILL_COLOUR = "#c7ffca";
+
+    public static final String INPUT_CLAUSE_BACKGROUND_FILL_COLOUR = "#ddffdf";
+
+    public static final String OUTPUT_CLAUSE_BACKGROUND_FILL_COLOUR = "#e9ffea";
+
+    public static final String DESCRIPTION_COLUMN_BACKGROUND_FILL_COLOUR = "#f7f7f7";
+
+    public static final String EXPRESSION_COLUMN_BACKGROUND_FILL_COLOUR = "#f7f7f7";
+
+    public static final String UNDEFINED_EXPRESSION_COLUMN_BACKGROUND_FILL_COLOUR = "#f7f7f7";
+
+    public static final String LITERAL_EXPRESSION_COLUMN_BACKGROUND_FILL_COLOUR = "#f7f7f7";
+
+    public static final String RELATION_BACKGROUND_FILL_COLOUR = "#f7f7f7";
+
+    public static final String FONT_FAMILY_HEADER = "Open Sans, Helvetica, Arial, sans-serif";
+
+    public static final String FONT_FAMILY_LABEL = "Open Sans, Helvetica, Arial, sans-serif";
+
+    public static final String FONT_FAMILY_EXPRESSION = "Courier New";
+
+    public static final double SELECTOR_STROKE_WIDTH = 2.0;
+
+    public static final double STROKE_WIDTH = 1.0;
+
+    public static final int FONT_SIZE = 10;
+
+    @Override
+    public String getName() {
+        return "Scenario Simulation Editor";
+    }
+
+    @Override
+    public MultiPath getSelector() {
+        return new MultiPath().setVisible(false);
+    }
+
+    @Override
+    public Rectangle getCellSelectorBorder() {
+        return new Rectangle(0, 0)
+                .setStrokeColor(HOVER_STATE_STROKE_COLOUR)
+                .setStrokeWidth(SELECTOR_STROKE_WIDTH);
+    }
+
+    @Override
+    public Rectangle getCellSelectorBackground() {
+        return new Rectangle(0, 0).setVisible(false);
+    }
+
+    @Override
+    public Rectangle getHeaderBackground(final GridColumn<?> column) {
+        return new Rectangle(0, 0)
+                .setFillColor(BACKGROUND_FILL_COLOUR);
+    }
+
+    @Override
+    public Rectangle getHeaderLinkBackground(final GridColumn<?> column) {
+        return new Rectangle(0, 0)
+                .setFillColor(ColorName.LIGHTGRAY);
+    }
+
+    @Override
+    public MultiPath getHeaderGridLine() {
+        return new MultiPath()
+                .setStrokeColor(GRID_STROKE_COLOUR)
+                .setStrokeWidth(STROKE_WIDTH)
+                .setVisible(true);
+    }
+
+    @Override
+    public Text getHeaderText() {
+        return new Text("")
+                .setFillColor(ColorName.BLACK)
+                .setFontSize(FONT_SIZE)
+                .setFontFamily(FONT_FAMILY_HEADER)
+                .setTextUnit(TextUnit.PT)
+                .setListening(false)
+                .setTextBaseLine(TextBaseLine.MIDDLE)
+                .setTextAlign(TextAlign.CENTER);
+    }
+
+    @Override
+    public Rectangle getBodyBackground(final GridColumn<?> column) {
+        final Rectangle background = new Rectangle(0, 0);
+       /* if (column instanceof NameColumn) {
```

Comment:
If these are not needed, please remove them.

Reply:
Fixed with
[DROOLS-2672] Removing commented code from BaseExpressionGridTheme